<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>回显模式</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="le_1"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_out1">
          <property name="text">
           <string>输出内容</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_normal">
          <property name="text">
           <string>正常模式</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_password">
          <property name="text">
           <string>密码模式</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_noecho">
          <property name="text">
           <string>无回显</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_pe">
          <property name="text">
           <string>失去焦点变密码模式</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>对齐方式</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="le_2"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="btn_l">
          <property name="text">
           <string>左对齐</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_c">
          <property name="text">
           <string>居中对齐</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_r">
          <property name="text">
           <string>右对齐</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>读写控制</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="le_3"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_out3">
          <property name="text">
           <string>输出内容</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="btn_readwrite">
          <property name="text">
           <string>可读可写</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_readonly">
          <property name="text">
           <string>只读</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_disable">
          <property name="text">
           <string>禁能</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_enable">
          <property name="text">
           <string>使能</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>格式控制</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLineEdit" name="le_4"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QPushButton" name="btn_masknone">
          <property name="text">
           <string>无控制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_maskphone">
          <property name="text">
           <string>手机号码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_maskkey">
          <property name="text">
           <string>密钥</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>信号槽</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>textChange信号：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLineEdit" name="le_51">
          <property name="placeholderText">
           <string>被加数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_52">
          <property name="placeholderText">
           <string>加数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_53">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>和</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>editingFinished信号：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLineEdit" name="le_54">
          <property name="placeholderText">
           <string>被减数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_55">
          <property name="placeholderText">
           <string>减数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_56">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>差</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
